### PR TITLE
WIP: Fuse online from docker hub image

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.14.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by calebevans** [comment]
@smatula to fix the error you messaged me about, you'll have to add an entry for the scenario in the `core-services/prow/02_config/_plugins.yaml` file. An example of that can be found here: https://github.com/openshift/release/pull/40278/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07b

**Comment by smatula** [comment]
/pj-rhearse periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse pull-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-images periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

**Comment by smatula** [comment]
/pj-rehearse pull-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-images periodic-ci-syndesisio-syndesis-qe-1.15.x-fuse-online-ocp4.14-lp-interop-fuse-online-interop-aws

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 2Gi
+tests:
+- as: fuse-online-interop-aws
+  cron: 0 6 23 11 *
```
Comment (ascerra) [inline ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:43]:
```suggestion
  cron: 0 6 * * 1
```

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
```
Comment (ascerra) [inline ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:50]: We no longer need to use the jira-epic LPTOCPCI-290 now that Aditya has created the firewatch ticket tracking dashboard https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12353648

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.14-lp-interop.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+tests:
+- as: fuse-online-interop-aws
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
```
Comment (ascerra) [inline ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.14-lp-interop.yaml:50]: We no longer need to use the jira-epic LPTOCPCI-290 now that Aditya has created the firewatch ticket tracking dashboard https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12353648

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |-
+    FROM quay.io/fuse_qe/syndesisqe-tests:1.14.x
```
Comment (ascerra) [inline ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:13]: Do we still need to be testing this older version?

Comment (ascerra) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:13]: This is touching on the same topic being discussed here
https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1689177285378299

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 1 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:43]: @ascerra what do we set when we no longer want it to run

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 4 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml:13]: @ascerra Was originally developed and tested here.  Just yesterday was asked to move to 1.15.  Want to keep for historical purposes n -1. I was expecting to rm when 1.16 comes out.  Also, was unsure if they wanted to run both. Asked Badre and he indicated to only run the latest.

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 3 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.14-lp-interop.yaml]: Removed jira-epic from firewatch configurations

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 2 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml]: Removed jira-epic from firewatch configurations.

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 4 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml]: removed

File: ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml (hunk 1 above)
Comment (smatula) [reply ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml]: removed

**Pull request closed without merging**